src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="index.html"
+      method="post"
+      class="big__bar"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="big_search"
+      >
+      <label for="big_search_input"></label>
+      <input
```

Comment:
add name attributes to inputs

Reply:
for example?i dont understand

Reply:
just name attribute like
`name="bigSearch"`

read about attributes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes